app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  def update_likes_count
+    post.update(likes_counter: post.likes.size)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] We can make use of some model callback functions such as __after_save__.

```ruby
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:likes_counter) # using the increment function will increment the likes_counter without calling on size function.
  end
```